Windows.Devices.WiFi/WiFiAdapter.cs

```diff
@@ -133,6 +133,22 @@ public WiFiConnectionResult Connect(WiFiAvailableNetwork availableNetwork, WiFiR
             }
         }
 
+        /// <summary>
+        /// Connect this Wi-Fi device to the specified network (using SSID string), with the specified passphrase and reconnection policy.
+        /// </summary>
+        /// <param name="ssid">Describes the Wi-Fi network to be connected.</param>
+        /// <param name="reconnectionKind">Specifies how to reconnect if the connection is lost.</param>
+        /// <param name="passwordCredential">The passphrase to be used to connect to the access point.</param>
+        /// <returns>
+        /// On successful conclusion of the operation, returns an object that describes the result of the connect operation.
+        /// </returns>
+        public WiFiConnectionResult Connect(string ssid, WiFiReconnectionKind reconnectionKind, string passwordCredential)
```

Comment:
Seems more straightforward when using to have it (ssid, passwordCredential, reconnectionKind). 
Can you please change the order?

Reply:
I agree that the param order is confusing but that's the way the original Connect() method is.
We probably shouldn't change the original Connect() param order since it's a public method and we would break user code all over the world.
Should we leave the new Connect() param order consistent with the original?

Reply:
Got you... you're right. Let's leave it as it is then.